src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -770,15 +768,14 @@ private HearingOptions buildHearingOptions(Map<String, Object> pairs, String hea
 
         String languageType = isLanguageInterpreterRequired ? findLanguageTypeString(pairs) : null;
 
-        String wantsToAttend = hearingType != null && hearingType.equals(HEARING_TYPE_ORAL) ? YES_LITERAL : NO_LITERAL;
+        String wantsToAttend = (hearingType.equals(HEARING_TYPE_ORAL)) ? YES_LITERAL : NO_LITERAL;
```

Comment:
Why remove the null check here?

Reply:
Sorry, my wrong. I add it back and update the branch. Thanks for reminding.